services/filesstore/filesstore.go

```diff
@@ -27,6 +27,7 @@ type FileBackend interface {
 	RemoveFile(path string) *model.AppError
 
 	ListDirectory(path string) (*[]string, *model.AppError)
+	ListAll(path string) ([]string, *model.AppError)
```

Comment:
Maybe document what exactly this returns.

Reply:
After testing in S3 we realized this is unnecessary. Will be removing this extra implementation now.

Reply:
Removed